docs/develop/node/community/node-community-updates.md

```diff
@@ -0,0 +1,51 @@
+---
+id: node-community-updates
+title: Node Community Updates
+sidebar_label: Node Community Updates
+description: NEAR Node and Validator Community Updates
+---
+
+## NEAR Validator Community Channels
+
+NEAR Protocol will communicate with validators using these channels:
+
+1. Releases of `nearcore` are on Github repository of [`nearcore`](https://github.com/near/nearcore/issues).
+
+2. Bugs and feature enhancements are tracked in the Github project tracker [`Node Experience`](https://github.com/orgs/near/projects/18). Node operators are welcome to create new issues for features and bugs, and to add these issues into the [Github project tracker](https://github.com/orgs/near/projects/18).
+    - **New feature / Enhancement requests:**  
+        - Please create a [`Github issue`](https://github.com/near/nearcore/issues)
+        - Tag `nodeX`
+        - Filter them through `Incoming Requests` column in the Github project tracker [`Node Experience`](https://github.com/orgs/near/projects/18), where they will be groomed and moved into `Backlog` based on priority
+    - **Bug reports:**
+        - Please create a [`Github issue`](https://github.com/near/nearcore/issues)
+        - Tag `nodeX` and `Bug`
+        - Put it at the top of the `Incoming Requests` column for more immediate attention in the Github project tracker [`Node Experience`](https://github.com/orgs/near/projects/18)
```

Comment:
non-NEAR members cannot label issues or move issues on those boards. We need to create GitHub templates with the pre-set labels to let people create the issues with the specific labels.